content/chainguard/migration-guides/migrating-to-chainguard-images.md

```diff
@@ -0,0 +1,130 @@
+
+| Command Description          | Debian-based Dockerfile | Wolfi-based Equivalent |
+|------------------------------|-------------------------|------------------------|
+| Install a package            | `apt install`           | `apk add`              |
+| Remove a package             | `apt remove`            | `apk del`              |
+| Update package manager cache | `apt update`            | `apk update`           |
+
+Our [Debian Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/debian-compatibility/) page has a table listing common tools and their corresponding package(s) in both Wolfi and Debian distributions. For Ubuntu-based Dockerfiles, check our [Ubuntu Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/ubuntu-compatibility/) page.
+
+## Migrating from Red Hat UBI Dockerfiles
+If you are coming from a Red Hat UBI (Universal Base Image) Dockerfile, you'll also need to adapt some of your commands to be compatible with the apk ecosystem. Wolfi uses BusyBox utilities, which offer a smaller footprint compared to GNU coreutils in Red Hat images. Our [Red Hat Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/debian-compatibility/) page has a table listing common tools and their corresponding package(s) in both Wolfi and Red Hat distributions.
+
+If you are coming from a Red Hat UBI based Dockerfile, you'll need to adapt some of your commands to be compatible with the apk ecosystem:
+
+| Command Description          | Red Hat UBI Dockerfile | Wolfi-based Equivalent |
+|------------------------------|------------------------|------------------------|
+| Install a package            | `yum install`          | apk add                |
+| Remove a package             | `yum remove`           | apk del                |
+| Update package manager cache | `yum makecache`        | apk update             |
+
+## Migrating from Alpine Dockerfiles
+If your Dockerfile is based on Alpine, the process for migrating to Chainguard Images should be more straightforward, since you're already using `apk` commands. Wolfi packages typically match what is available in Alpine, with some exceptions. For instance, the Wolfi busybox package is slimmer and doesn't include all tools available in Alpine's busybox. Check the [Alpine Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/alpine-compatibility/) page for a list of common tools and their corresponding packages in Wolfi and Alpine.
+
+## Searching for Packages
+Packages from Debian and other base distributions might have a different name in Wolfi. To search for packages, log into an ephemeral container based on `cgr.dev/chainguard/wolfi-base`:
+
+```shell
+docker run -it --rm --entrypoint /bin/sh cgr.dev/chainguard/wolfi-base
+```
+
+Then, run `apk update` to update the local apk cache with latest Wolfi packages:
+
+```shell
+# apk update
```

Comment:
```suggestion
apk update
```
These commands all work but it makes sense to me to break up these commands from their expected outputs

Reply:
`#` is root's prompt, so this shows nearly exactly what you'd expect to see. A lot of people do this with `#` or`$` to indicate running as root or as a user. I wish they wouldn't.

This is a small bugbear of mine, when you view the rendered file there is a "copy" shortcut button on the command's text box and it copies _everything_ . Including the `#` and command response. Which makes the copy button useless, because you can't paste commands prefixed with # and pasting the result of someone else's command into your terminal just makes a mess.

Further down where we `apk search` in one commands block and the results are in the next is better IMHO (or would be if the `apk search` didn't have a `#` !) Because you can copy _just_ the command, without the response.

Rather than repeating the command from the preamble in the example, my suggestion would be something like:

> 
> Then, run:
> ```
> apk update
> ```
> to update the local cache, If the update is successful, you'll see a response like:
> ```
> fetch https:... 
> ......
> OK 1234 distinct packages available
> ```

For ultimate "perfection" could we remove the "copy" button from the response text blocks? If someone wants to copy the response, they can select/copy manually. (This is strictly in the "nice to have" category if it is hard).